torch/_torch_docs.py

```diff
@@ -2776,6 +2776,29 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.hypot,
+           r"""
+hypot(input, other, out=None) -> Tensor
+
+Given the legs of a right triangle, return its hypotenuse.
+
+Equivalent to calculating pointwise :math:`\sqrt{x^2 + y^2}`.
```

Comment:
This should probably use a treatment more like here:

https://github.com/pytorch/pytorch/blob/a7c04774f992c20087ff93e4858bf8f5c7ec01f4/torch/_torch_docs.py#L140

`x` and `y` don't refer to anything in this context, so you'll want to reference `input` and `other`.